lib/coverage-middleware.js

```diff
@@ -18,23 +19,24 @@ function reporterForName(reporterName) {
   return require(path.join(__dirname, 'reporters', reporterName + '-reporter'));
 }
 
-// TODO handle multiple reporters
-function loadReporter(blanketOptions) {
-  var reporterName = blanketOptions.cliOptions.reporters[0] || 'null';
-
-  var OutputReporter = reporterForName(reporterName);
+function loadReporter(blanketOptions, name) {
+  var OutputReporter = reporterForName(name);
   var reporter = new OutputReporter(blanketOptions);
   return reporter;
 }
 
 module.exports = function(options) {
 
-
   var blanketOptions = loadBlanketOptions(options);
-  var reporter = loadReporter(blanketOptions);
+  var reporters = _.map(blanketOptions.cliOptions.reporters, function (reporter) {
```

Comment:
I get that lodash is present but `blanketOptions.cliOptions.reporters.map(function(reporter) ...` would work - same for the .forEach below